ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/event-handlers.j2

```diff
@@ -9,7 +9,40 @@
 !
 event-handler {{ handler.name }}
 {%         if handler.trigger is arista.avd.defined %}
+{%             if handler.trigger is arista.avd.defined("on-maintenance")
+                  and handler.trigger_on_maintenance is arista.avd.defined
+                  and handler.trigger_on_maintenance.operation is arista.avd.defined
+                  and handler.trigger_on_maintenance.stage is arista.avd.defined %}
+{%                 set trigger_on_manitenance_cli =  "trigger " ~ handler.trigger ~ " " ~ handler.trigger_on_maintenance.operation %}
+{%                 if handler.trigger_on_maintenance.bgp is arista.avd.defined %}
+{%                     set trigger_on_manitenance_cli = trigger_on_manitenance_cli ~ " bgp " %}
```

Comment:
is this valid if neither neighbor_* or peer_group_name is set? If not we need to add more conditions on this `if`

Reply:
no its not valid  if neither neighbor_* or peer_group_name is set

Reply:
if neither of the keys is defined then it won't go inside
 `{% if handler.trigger_on_maintenance.bgp is arista.avd.defined %}`
 if some key is defined other than neighbor_* and peer_group it will give `unexpected keys error` 
 
 Am I understanding your concern correctly here?

Reply:
you could have an empty `bgp` dict, so no keys in it.

Reply:
changed it to bgp_peer